src/Button.svelte

```diff
@@ -38,7 +38,7 @@
   <a
     {...$$restProps}
     class={classes}
-    {disabled}
+    class:disabled
```

Comment:
`disabled` is a real prop on HTML buttons, why the switch to a class name?

Reply:
not sure what you mean
i removed the disabled attribute of the anchor tag, as it does not have a disabled attribute.

Reply:
Ah thanks for the clarification, missed that